### ISSUE TITLE
influxes receiver might need more endpoints

### ISSUE THREAD

**Issue by skandragon** [open-telemetry/opentelemetry-collector-contrib#33817]
### Component(s)

receiver/influxdb

## Description

I'm trying to point my Proxmox cluster to the influxes receiver, but it fails to add.  It attempts to do a `GET /health` which is most likely some sort of initial probe test.  It then fails to add the metric feed.

### Collector version

v0.103.0

## Environment
Proxmox

**Comment by skandragon** [comment]
@open-telemetry/collector-contrib-triagers

I'm all for "auto closing issues" when the answer is "we won't do this, but feel free to make a PR" but if no response is given, it makes it hard to know what to do next here.

**Comment by skandragon** [comment]
@jacobmarble

**Comment by atoulme** [comment]
Sorry @skandragon this is all automated management. @jacobmarble any chance you can weigh in?

@skandragon would you be interested to provide a fix or help create a test reproducing the issue?

**Comment by skandragon** [comment]
> @skandragon would you be interested to provide a fix or help create a test reproducing the issue?

Quite possibly.  Let me add it to the list :)

**Comment by jacobmarble** [comment]
I'm sorry I missed this.

There aren't a lot of details to go on here. Does the /health probe fail?

**Comment by skandragon** [comment]
Yes, /health is returning 404 if memory serves.  I have not poked this for a while so that is from memory.  I know the /health probe was not returning what Proxmox's UI wanted to see at least.
